### PR TITLE
Release 4.1.1

### PR DESCRIPTION
**Pull request by ashie**
Remaining issues:

* [x] Update Fluentd to 1.12.3
* [x] Test and merge #289
* [x] Check td command #293
* [x] Waiting RubyInstaller 2.7.3
* [x] I'm considering merge #285 before this, but not yet detemined (may be postpone to 4.2.0)
* [x] fluent-plugin-s3 may be updated to the next version (1.6.0)
* [x] Validate module versions
* [x] Update CHANGELOG.md
* [x] Check built packages

TODO for 4.2.0 (don't include into 4.1.1)

* ~~fluent-plugin-prometheus_pushgateway should be updated to use with fluent-plugin-prometheus 2.0.0~~
  Done, but we won't bundled it in td-agent 4.1.1 because it's mainly a bug fix release.

### CODE REVIEW

**Comment by kenhys** [comment]
How about https://github.com/fluent-plugins-nursery/td-agent-builder/pull/289?
Is it merged for 4.1.1, or postponed to 4.2.0?

**Comment by ashie** [comment]
> How about #289?

I'll test it on my environments.
It should be merged to 4.1.1 if it works well.

**Comment by kenhys** [comment]
> Update to ruby-2.5.9, 2.6.7, 2.7.3 and 3.0.1 https://github.com/oneclick/rubyinstaller2/commit/cece6dcf649513b441cf091dcd0c3edcb230403a

Still work in progress (ruby installer)

**Comment by ashie** [comment]
We should also include a [patch](https://github.com/fluent/fluentd/issues/3327) for https://github.com/fluent/fluentd/issues/3327

**Comment by cosmo0920** [comment]
> > Update to ruby-2.5.9, 2.6.7, 2.7.3 and 3.0.1 [oneclick/rubyinstaller2@cece6dc](https://github.com/oneclick/rubyinstaller2/commit/cece6dcf649513b441cf091dcd0c3edcb230403a)
> 
> Still work in progress (ruby installer)

Perhaps, they will be released in the next week:
https://github.com/oneclick/rubyinstaller2/issues/221#issuecomment-818074516

RubyInstaller is also affected coronavirus pandemic.... :cry:

**Comment by cosmo0920** [comment]
@ashie Should I need to publish this patch https://github.com/cosmo0920/win32-api/pull/55 as win32-api v1.10.0?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,65 @@
 # Treasure Agent 4 changelog
 
+## Release v4.1.1 - 2020/04/30
```
Comment (tjstansell) [inline CHANGELOG.md:3]: wrong year.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+* fluent-plugin-flowcounter-simple 0.1.0
+* fluent-diagtool v1.0.1
+
+On Windows
+
+* winevt_c v0.9.2 (update)
+* fluent-plugin-parser-winevt_xml v0.2.2
+* fluent-plugin-windows-eventlog v0.8.0
+
 ## Release v4.1.0 - 2020/02/24
```
Comment (tjstansell) [inline CHANGELOG.md:63]: wrong year here too.

File: CHANGELOG.md (hunk 1 above)
Comment (ashie) [reply CHANGELOG.md:3]: Oooops, thanks pointing out it!

**Comment by toru-takahashi** [comment]
I've published v4.1.1
Thanks!

**Comment by ashie** [comment]
Thanks! I'll update the release date & merge this PR.

**Pull request merged**